src/jsSSE.php

```diff
@@ -114,7 +133,14 @@ private function output(string $content): void
     public function sendBlock(string $id, string $data, string $name = null): void
     {
         if (connection_aborted()) {
-            exit();
+            if ($this->onAborted && is_callable($this->onAborted)) {
+                call_user_func($this->onAborted, $this, $this->param);
```

Comment:
There should be hook instead. No `onAborted` and no `onAborted ` not `params` props.

Reply:
make sense. done!